Python/PRP/Animation/pyLeafController.cpp

```diff
@@ -64,28 +68,27 @@ static PyObject* pyLeafController_setKeys(pyLeafController* self, PyObject* args
             PyErr_SetString(PyExc_TypeError, "setKeys expects a list of hsKeyFrames and an int");
             return NULL;
         }
-        ((pyKeyFrame*)itm)->fPyOwned = false;
```

Comment:
Should this have been removed?  I would expect the controller to take ownership of the keys...